source/integrations/cloud-bot-accounts.rst

```diff
@@ -153,7 +153,7 @@ If you find yourself unable to edit posts as a bot, check the following:
 1. Instead of using a slash command to respond directly, use an an API call for the initial interaction with a user to enable message edits.
 2. If your system is using `advanced permissions <https://docs.mattermost.com/cloud/cloud-user-management/advanced-permissions.html>`_, then post edits could be disabled for users.
 
-If neither of the above help resolve your concern, you also have the option to choose what role the bot account has. If the **System Admin** role is chosen, then they can update any posts in the system, along with other System Admin permissions. Note that giving the **System Admin** role to a bot account enables them with other System Admin privileges so this should be done with care.
+If neither of the above help resolves your concern, you also have the option to choose what role the bot account has. If the **System Admin** role is chosen, they can update any posts in the system and other System Admin permissions. Note that giving the **System Admin** role to a bot account enables them with other System Admin privileges, so this should be done with care.
```

Comment:
```suggestion
You also have the option to choose what role the bot account has. If the **System Admin** role is chosen, they can update any posts in the system and other System Admin permissions. Note that giving the **System Admin** role to a bot account enables them with other System Admin privileges, so this should be done with care.
```

Reply:
Yeah this is more apt. Less writing with more impact.